gobblin-api/src/main/java/org/apache/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -117,6 +117,11 @@
   public static final String HELIX_JOB_TIMEOUT_SECONDS = "job.timeout.seconds";
   public static final String DEFAULT_HELIX_JOB_TIMEOUT_SECONDS = "10800";
 
+  public static final String HELIX_TASK_TIMEOUT_ENABLED_KEY = "task.timeout.enabled";
+  public static final boolean DEFAULT_HELIX_TASK_TIMEOUT_ENABLED = false;
```

Comment:
I think we should remove this key. It is deceptive since helix has task timeout set to 60 minutes by default. So we should just remove this key and have an appropriate default timeout set by gobblin. We can default to the same 60  minutes.